### ISSUE TITLE
[104] Announce ways to access back layer of backdrop

### ISSUE THREAD

**Issue by michdud** [material-components/material-components-flutter-codelabs#86]
The menu button and top layer of the backdrop allow users to open the back layer of the backdrop, but are not announced as such with TalkBack.
